### ISSUE TITLE
Add tab order to GUI elements

### ISSUE THREAD

**Issue by ObaltzerS** [Hendricks-Laboratory/OT2Control#95]
The GUI currently does not have any support for tab order or screen readers, adding this would make it more accessible and usable.
